generic_server.cc

```diff
@@ -14,11 +14,101 @@
 #include <seastar/coroutine/parallel_for_each.hh>
 #include <seastar/core/reactor.hh>
```

Comment:
> There will be a simplification here as we'll treat disc IO (if there is any)
as cpu work.

Yes. I think it's also okay to reduce the concurrency of new connections to limit disk I/O.